packages/dataviews/src/types.ts

```diff
@@ -164,3 +202,78 @@ export interface ViewList extends ViewBase {
 }
 
 export type View = ViewList | ViewBase;
+
+export interface ActionBase< T extends Item > {
```

Comment:
This is the third PR that adds Action type, can we remove that from the PR for now. Just to avoid multiple PRs conflicting on the same thing.

Reply:
I think we should probably keep the PR about existing types and not add new ones. I prefer if we add new ones in their own PRs if you don't mind.

Reply:
The Action types are already on trunk, so this needs a rebase.

Reply:
@youknowriad I'll remove them and rebase.

Reply:
I'll will add the generic arguments if that's okay.